src/main/java/org/jabref/gui/libraryproperties/contentselectors/ContentSelectorViewModel.java

```diff
@@ -102,7 +102,7 @@ BooleanBinding isNoKeywordSelected() {
     }
 
     void showInputFieldNameDialog() {
-        dialogService.showInputDialogAndWait(Localization.lang("Add new field name"), Localization.lang("Field name:"))
+        dialogService.showInputDialogAndWait(Localization.lang("Add new field name"), Localization.lang("Field name") + ":")
```

Comment:
Please don't do this! you never know other non latin languages handle this!

Reply:
A) The existing code already has these things

B) all translations have the : at the end - or omit it. 

Example:

![image](https://github.com/JabRef/jabref/assets/1366654/1063b46b-f99b-4c4a-a8e9-1168786aa352)


I think, all of them are wrong ^^

Example for this:

![image](https://github.com/JabRef/jabref/assets/1366654/ecccc677-28c1-4e12-a426-370999ae25dd)

The : should be removed as the label formatting itself should indicate that it is a label - and not the `:`